sanskrit_parser/util/normalization.py

```diff
@@ -12,9 +12,10 @@
 logger = logging.getLogger(__name__)
 
 deletions = re.compile("[\u200b\u200c\u200d,'-;().?!\"0123456789{}#\r]")
+rakarantas = ['punaH', 'antaH']
```

Comment:
Won't this list be too long due to the way any R-kArAnta sambodhana is stored ? pitar, mAtar, bhrAtar, ....

Reply:
What are our options here?

Reply:
Since this specific case is related to tag lookup portion only, can we push it into the tag module (e.g. inriaxmlwrapper) and follow your earlier suggestion of trying to lookup a word with trailing 's' and if that doesn't work, try with an 'r'? We could have a `strict` parameter control whether the tag lookup does this or not.

Reply:
Might be easier to move the visarga replacment out into separate functions then? We could have a replace_visarga_s and replace_visarga_r, and explicitly have tag lookup try one after the other? The default normalization function can not worry about visargas.

Which brings us to denormalization. Shouldn't denormalization handle rakarAntas?

Reply:
Separate functions with a parameter to control whether each of them is applied sounds like a good idea! 

Yes. I think it should. Good catch.